.github/workflows/test.yml

```diff
@@ -60,6 +60,11 @@ jobs:
         with:
           workspaces: dask_planner
           shared-key: test
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v1
+        with:
+          version: '3.x'
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Would remove this and instead add `libprotobuf=3` to the conda dev environments so that users don't need to install it manually

```suggestion
```